components/example.js

```diff
@@ -1,75 +1,77 @@
-const { Component } = require("../component")
+const { Component } = require('../component');
 
-const ID = "example";
+const ID = 'example';
 
 class Example extends Component {
     constructor() {
         super(ID);
-        this.addCommand("\\+win$", this.win);
-        this.addCommand("\\+countdown (\\d+)$", this.countdown);
-        this.addCommand("\\+countdown$", () => this.countdown(5));
-        this.addCommand("\\+delay (\\d+) (.*)$", this.delay);
-        this.addCommand("\\+spam (\\d+) (.*)$", this.spam);
-        this.addCommand('\\+alarm \\[(.*)\\] (.*)$', this.alarm);
-        this.addCommand('\\+play (.*)$', this.playAudio);
-        this.addCommand('\\+play$', () => this.playAudio("sample.mp3"));
-        this.addCommand('\\+coinflip$', () => this.coinflip("", ""));
-        this.addCommand('\\+coinflip (\\S+) (\\S+)$', this.coinflip);
-        this.addCommand('\\+coinflip "(.*)" "(.*)"$', this.coinflip);
+        this.addCommand(/\+win$/, this.win);
+        this.addCommand(/\+countdown (\d+)$/, this.countdown);
+        this.addCommand(/\+countdown$/, () => this.countdown(5));
+        this.addCommand(/\+delay (\d+) (.*)$/, this.delay);
+        this.addCommand(/\+spam (\d+) (.*)$/, this.spam);
+        this.addCommand(/\+alarm \[(.*)] (.*)$/, this.alarm);
```

Comment:
Is this missing a backslash before the ']' ?

Reply:
Surprisingly `]` only needs to be escaped if you already used `[`